target/scripts/startup/misc-stack.sh

```diff
@@ -33,6 +33,8 @@ function _misc_save_states
     [[ ${ENABLE_FAIL2BAN} -eq 1 ]] && FILES+=('lib/fail2ban')
     [[ ${ENABLE_FETCHMAIL} -eq 1 ]] && FILES+=('lib/fetchmail')
     [[ ${ENABLE_POSTGREY} -eq 1 ]] && FILES+=('lib/postgrey')
+    [[ ${ENABLE_RSPAMD} -ne 1 ]] && FILES+=('lib/rspamd')
+    # [[ ${ENABLE_RSPAMD} -ne 1 ]] && FILES+=('lib/redis')
```

Comment:
Intentional to leave commented out?

Reply:
Fair enough. I'm happy to leave as-is then for the time being :+1:

Reply:
Redis is an in-memory data store. I think there is no reason to persist its data.